### ISSUE TITLE
Add link to FAQ for words-length error message upon Secret Key entry

### ISSUE THREAD

**Issue by timstackblock** [blockstack/stacks-wallet#641]
## SCENARIO 1

### - There is absolutely no issue and the migration from the old wallet to the new wallet is seamless.  

<img width="349" alt="Screen Shot 2021-02-25 at 12 25 25 PM" src="https://user-images.githubusercontent.com/49165468/109191908-a1ce0300-7764-11eb-9201-f0f1b66dcc62.png">

<img width="1402" alt="Screen Shot 2021-02-25 at 11 36 04 AM" src="https://user-images.githubusercontent.com/49165468/109191261-fd4bc100-7763-11eb-9c5b-7b1df8653f1b.png">

<img width="1410" alt="Screen Shot 2021-02-25 at 11 36 22 AM" src="https://user-images.githubusercontent.com/49165468/109191262-fde45780-7763-11eb-8448-5ca353b02c86.png">

<img width="938" alt="Screen Shot 2021-02-25 at 11 48 41 AM" src="https://user-images.githubusercontent.com/49165468/109191270-ff158480-7763-11eb-99cd-52e6584e1ee5.png">

<img width="938" alt="Screen Shot 2021-02-25 at 11 48 49 AM" src="https://user-images.githubusercontent.com/49165468/109191271-ff158480-7763-11eb-8de1-a754569e3459.png">

<img width="1019" alt="Screen Shot 2021-02-25 at 11 51 53 AM" src="https://user-images.githubusercontent.com/49165468/109191272-ff158480-7763-11eb-8bd4-f8f43a703d3a.png">

<img width="1007" alt="Screen Shot 2021-02-25 at 11 52 31 AM" src="https://user-images.githubusercontent.com/49165468/109191275-ffae1b00-7763-11eb-87ef-2b41d2caaaa0.png">

_________________________________________________________________________________________________________________________________
### Please see the images below for more description.

<img width="1370" alt="Screen Shot 2021-02-25 at 12 06 42 PM" src="https://user-images.githubusercontent.com/49165468/109191278-0046b180-7764-11eb-94cd-446d7df8a9a8.png">
<img width="723" alt="Screen Shot 2021-02-25 at 12 51 29 PM" src="https://user-images.githubusercontent.com/49165468/109195354-68979200-7768-11eb-8209-c57ea994f8d9.png">

<img width="1397" alt="Screen Shot 2021-02-25 at 12 09 36 PM" src="https://user-images.githubusercontent.com/49165468/109191279-0046b180-7764-11eb-99a1-ed7f583ddd27.png">
<img width="723" alt="Screen Shot 2021-02-25 at 12 51 29 PM" src="https://user-images.githubusercontent.com/49165468/109195210-3d14a780-7768-11eb-8a40-099f28c05efe.png">

<img width="768" alt="Screen Shot 2021-02-25 at 12 35 03 PM" src="https://user-images.githubusercontent.com/49165468/109193125-0b9adc80-7766-11eb-9ab8-692389c3d29d.png">

<img width="945" alt="Screen Shot 2021-02-25 at 12 11 52 PM" src="https://user-images.githubusercontent.com/49165468/109191280-00df4800-7764-11eb-9230-dd47bf7b5333.png">

**Comment by jeffdomke** [comment]
Thanks @timstackblock

Also context what happens when I try to enter 12, 13, or 26 words into v4? An error in all cases? Could it take those inputs in theory to generate the correct address? @kyranjamie

**Comment by timstackblock** [comment]
It doesn't error as long as we go over the 12 or 24 word expected input. If I enter 13 or 25 words it ignores the last word and it just opens the 12 word or 24 word seed phrase wallet.

**Comment by timstackblock** [comment]
Basically we never planned for that input so it doesn't recognize it.

**Comment by timstackblock** [comment]
@markmhx @andresgalante this is kind of a big deal we should try to get this into a sprint

**Comment by markmhx** [comment]
It's indeed important to support 13-word, 25-word etc phrases here without treating them as 12, 24, etc.

@jeffdomke to be clear, though, you were wondering if key values entered strictly as 12 words could be interpreted differently in the new wallet vs. the old, correct? Or is it possible that you had this sort of passphrase scenario in mind as well?

**Comment by kyranjamie** [comment]
I suggest we just include a non-valid bip39 option you can enable, off by default

**Comment by GinaAbrams** [comment]
Came here to share from forum user Bruce: 

Tip:
The Stacks Wallet can be used with only 12 and 24-word Secret Keys

Can you provide a tool like this?
https://iancoleman.io/bip39/#english

**Comment by jeffdomke** [comment]
@markmhx I wasn't suggesting a feature or type of support. I was wondering if the following is happening: 

- User has 13 word seed created with Trezor
- User enters 13 words into software wallet
- Wallet correctly uses first 12 words to generate keypair
- User sees empty wallet address and then reports a problem to us

Does this happen or does error out?

**Comment by markmhx** [comment]
That appears to be the behavior @timstackblock reports here when 13 words are entered, in that it does _not_ error out.

By "you were wondering", I mean the other week when you asked us about this during a sync. IIRC you were inquiring about users entering _12_ words into the new wallet and whether they could end up with different addresses than with the old wallet. But I just wanted to confirm whether that was actually the same concern as the bug found here with 13 words.

**Comment by kyranjamie** [comment]
@jeffdomke  This does not happen, it'll will error out as it's not a valid phrase 

![image](https://user-images.githubusercontent.com/1618764/109327876-18e4c380-7859-11eb-89b0-74b2dc995776.png)

**Comment by markmhx** [comment]
I'm confused about this comment then? 🤔  https://github.com/blockstack/stacks-wallet/issues/641#issuecomment-786097806

**Comment by 0x20bf** [comment]
Now the question is:
There are users who used "trezor+passphrase" in the old wallet. In the new Stacks wallet, they are unable to restore their accounts using "24 words+passphrase".

I suggest to fix this issue as soon as possible, otherwise some users will not be able to get their crypto assets back.

**Comment by markmhx** [comment]
@0x20bf That's true it's important to fix this issue soon. Note, however, that meanwhile Trezor users should be able to migrate to Ledger for access to their crypto. This seems to affect entering Secret Key values directly only.

**Comment by kyranjamie** [comment]
> I'm confused about this comment then? 🤔 [#641 (comment)](https://github.com/blockstack/stacks-wallet/issues/641#issuecomment-786097806)

![image](https://user-images.githubusercontent.com/1618764/109377932-dcec4580-78ce-11eb-9c4b-028352e6188a.png)

Tested with 13 words too. @timstackblock can you elaborate, where did you see it ignoring the last word?

**Comment by 0x20bf** [comment]
> @0x20bf That's true it's important to fix this issue soon. Note, however, that meanwhile Trezor users should be able to migrate to Ledger for access to their crypto. This seems to affect entering Secret Key values directly only.

1. Trezor's "24 words+passphrase" cannot be restored to the Ledger wallet.
2, about Stacks wallet, there are two possible solution paths.
a. Enter "13/24 words+passphrase" to recover the wallet.
b. Enter a private key to recover an account.

**Comment by markmhx** [comment]
Is the Trezor passphrase not compatible for conversion into a Ledger passphrase along with the underlying seed phrase?

It seems they should be compatible, though I haven't tested it myself:

https://veriphi.io/en/blog/passphrase-tutorial
https://support.ledger.com/hc/en-us/articles/115005214529-Advanced-passphrase-security

**Comment by markmhx** [comment]
After discussing this as a team, we've decided to provide clearer guidance for Trezor users with passphrases for migrating to Ledger with their passphrases and not to support the entry of these passphrases directly into the wallet somehow without a hardware device.

I've modified this issue title accordingly. It's new aim should be to add simply a "Learn more" link to the end of the error message `The Stacks Wallet can be used with only 12 and 24-word Secret Keys`, which uses the following URL: 

https://www.hiro.so/questions/what-are-secret-keys-and-how-do-they-work#12%20vs.%2024-word%20Secret%20Keys

Users can find generic information there about both words length as well as specific guidance for Ledger and Trezor users with passphrases.
